tools/ktx/command_encode.cpp

```diff
@@ -138,13 +143,35 @@ int CommandEncode::main(int argc, char* argv[]) {
 
 void CommandEncode::OptionsEncode::init(cxxopts::Options& opts) {
     opts.add_options()
-        ("codec", "Target codec."
+        (kFormat, "KTX format enum that specifies the KTX file output format."
+                  " The enum names are matching the VkFormats without the VK_FORMAT_ prefix."
+                  " The VK_FORMAT_ prefix is ignored if present."
+                  "\nIt can't be used with --codec and is only valid for ASTC encoding."
```

Comment:
Delete "is only valid for ASTC encoding" as the start of the next sentence basically repeats this.

Reply:
I didn't think so. This says `--codec` and `--format` are mutualy exclusive options. The following says:

```
                  "\nThe format must be an ASTC format. When specified the ASTC encoder specific"
                  " options becomes valid."
```

This to me just says the format must be ASTC format, it doesn't talk about mutual exclusion with `--codec`

Reply:
Yes. I am not suggesting you remove "It can't be used with --codec", only the latter half of the sentence, which is what is repeated in the next sentence.

Reply:
I see. Happy to remove what you suggest.

Just adding my 2ps. I still think saying that the `--format` is only valid for ASTC encoding is still useful. Because here we are talking about the "option" itself while later we are talking about the "value" of the option. I am sure one can deduce this from "format needs to be only ASTC format means its not relevant for anything else bit" but thinking about someone who doesn't know this could trip on that.

Reply:
When I read it, it feels duplicative. The whole piece is talking about the --format option.

Change "The format must be" to "The value must be". I think then it is fine.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/952/commits/3645a7aff46fda0206c2bd32f4f38beaec26b26a

Reply:
"and is only valid for ASTC encoding" is still present. Please remove it.